lib/stm32/l0/timer.c

```diff
@@ -0,0 +1,31 @@
+/** @defgroup timer_file Timers
+ *
+ * @ingroup STM32F0xx
+ *
+ * @brief <b>libopencm3 STM32F0xx Timers</b>
```

Comment:
It's not really ok to just copy paste and not even review what you've copied.   At least, not for code you want to be merged upstream.  It's perfectly fine for your own working branch of course.